addon-test-support/@ember/test-helpers/-internal/helper-hooks.ts

```diff
@@ -0,0 +1,64 @@
+type Hook = (...args: any[]) => void | Promise<void>;
+type HookUnregister = {
+  unregister: () => void;
+};
+
+export const registeredHooks = new Map<string, Set<Hook>>();
```

Comment:
Once the unit tests have been updated to avoid mucking with this, we can stop exporting it at all:

```suggestion
const registeredHooks = new Map<string, Set<Hook>>();
```

Reply:
Done. I updated this in my local branch, so will ignore this suggestion.